rusk-wallet/src/bin/interactive.rs

```diff
@@ -199,15 +200,44 @@ fn menu_addr(wallet: &Wallet<WalletFile>) -> anyhow::Result<AddrSelect> {
             .add(AddrSelect::NewAddress, "New address")
     };
 
+    use rusk_wallet::SyncStatus;
```

Comment:
please add imports at the top of the file

Reply:
Move to the top